pkg/controllers/dynakube/extension/otel/statefulset.go

```diff
@@ -214,6 +215,13 @@ func buildContainerEnvs(dk *dynakube.DynaKube) []corev1.EnvVar {
 			},
 		},
 		},
+		{Name: envEECDStoken, ValueFrom: &corev1.EnvVarSource{
+			SecretKeyRef: &corev1.SecretKeySelector{
+				LocalObjectReference: corev1.LocalObjectReference{Name: dk.Name + consts.SecretSuffix},
```

Comment:
dk.Name + consts.SecretSuffix is often used, would it make sense to store it in a combined const instead?

Reply:
I don't understand. How would you store `dk.Name` in const? It's a variable value. But I can see if it makes sense to put it into a function.

Reply:
What's @luhi-DT probably wants to create a method of DK instance/type to calculate this string?

Reply:
correct

Reply:
dk.Name + consts.SecretSuffix is used 3 times, you could create one consts and use it instead, so if the secretname changes you only need to change it in one place